pages/common/rscript.md

```diff
@@ -0,0 +1,20 @@
+# Rscript
```

Comment:
Why is this page being added, even though it was added in another PR by another author? (#6686)

Reply:
not sure why this file is in 😞  i'll delete it